cmake/modules/AddSwift.cmake

```diff
@@ -1882,6 +1889,12 @@ function(_add_swift_executable_single name)
       LIBRARY_DIR ${SWIFT_LIBRARY_OUTPUT_INTDIR})
 
   target_link_libraries("${name}" ${SWIFTEXE_SINGLE_LINK_LIBRARIES} ${SWIFTEXE_SINGLE_LINK_FAT_LIBRARIES})
+
+  # On Windows, we need to link mincore.lib for various file system APIs used by Clang.
+  if ("${CMAKE_SYSTEM_NAME}" STREQUAL "Windows")
+    target_link_libraries("${name}" "mincore.lib")
```

Comment:
Yeah, this is weird - very weird. I've done some research - only `lib/AST` and `tools/SourceKit/lib/SwiftLang` need `mincore.lib`. This is because they directly link to `clangDriver.lib`